stacks-cli/data-scaffold-example.yml

```diff
@@ -0,0 +1,45 @@
+input:
+  log:
+    level: info
+    format: text
+    colour: true
+  directory:
+    working: stacks # Update: Target directory for the scaffolded project
+    export: "~"  # Update: Path to your Stacks CLI installation
+  business:
+    company: mycompany # Update: Used for resource naming
+    domain: mydomain  # Update: Used for environment & Terraform state key naming
+    component: none
```

Comment:
Should this be none? I know currently we do not use this, however there is a component field within our air-infrastructure-data-vars.yml which is not templated, currently it is hardcoded to 'de'. So should we instead be populating this value and add it as a template option?

Reply:
Updated file to suggest this is populated.
Draft PR for populating this value from the CLI: https://github.com/Ensono/stacks-azure-data/pull/171